onnxruntime/core/providers/cuda/rnn/cudnn_rnn_base.cc

```diff
@@ -5,44 +5,57 @@
 #include "cudnn_rnn_base.h"
 #include "rnn_impl.h"
 
+#if (CUDNN_MAJOR >= 8)
```

Comment:
Do we really need these guards ? IIRC we dropped CUDA 10.x support a while ago, and we can only use CuDNN 7.x and lower with CUDA 10.x. I feel there is no reason to introduce these guards in the source code ?